lib/console.js

```diff
@@ -61,11 +61,11 @@ Console.prototype.time = function(label) {
 
 
 Console.prototype.timeEnd = function(label) {
-  var time = this._times[label];
+  let time = this._times[label];
   if (!time) {
     throw new Error('No such label: ' + label);
   }
-  var duration = Date.now() - time;
+  let duration = Date.now() - time;
```

Comment:
I'm generally -0 on replacing `var` with `let` unless there's a distinct need for block-scoping.